open-policy-agent/service-account-mgmt/unique_serviceaccount_per_pod/constraint.yaml

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
@tommeramber @michaelkotelnikov

What is the reason to excludeNamespaces in each policy instead of applying the exclusion in a general-config file?

Reply:
Because there are policies that require cluster-wide vision, in which case by using a general config file/object - you will effectively prevent them from operating properly. So I've figured that a good compromise is to copy-paste the list of excluded Namespaces to the specific policy. 

@igorkchyts @michaelkotelnikov